components/brave_rewards/resources/tip/lib/host.ts

```diff
@@ -91,6 +92,19 @@ export function createHost (): Host {
       stateManager.update({ externalWalletInfo })
     },
 
+    tipFinished (result: number, amount: number) {
+      if (result === 0) {  // type::Result::LEDGER_OK
```

Comment:
We should get away from sending result integers from C++ into the front-end. We could either use a `boolean` to indicate success or failure, or we could split this into two separate handlers: perhaps `tipProcessed` and `tipFailed`. I'd probably lean toward having two separate handlers, since the `amount` is not meaningful in the failure case.

Reply:
Sure, fixed.